src/Microsoft.TestPlatform.ObjectModel/Resources/CommonResources.resx

```diff
@@ -121,7 +121,7 @@
     <value>The parameter cannot be null or empty.</value>
   </data>
   <data name="DisplayChosenSettings" xml:space="preserve">
-    <value>Test run will use DLL(s) built for framework {0} and platform {1}. Following DLL(s) do not match framework/platform settings.{2}Go to {3} for more details on managing these settings.
+    <value>The specified framework {0} and platform {1} is incompatible with the target framework/platform of test assemblies. Following DLL(s) do not match framework/platform settings.{2}Ensure that the target framework and platform of the test assemblies match the specified value. Go to {3} for more details on managing these settings.
```

Comment:
There are some redundancies and message looks very big.
How about. "The following DLL(s) do not match the specified framework {0} and platform {1} settings.{2}Ensure sure the test assemblies target the specified framework/platform settings. Go to {3} for more details on managing these settings."

/cc: @PBoraMSFT